doc/en/user/source/styling/sld/extensions/rendering-selection.rst

```diff
@@ -0,0 +1,306 @@
+.. _rendering_selection:
+
+Rendering Selection
+====================
+
+GeoServer provides a ``VendorOptions`` to define whether a particular element ``Rule``, ``FeatureTypeStyle`` or ``Symbolizer`` should be applied to a ``getLegendGraphic`` output or to a ``getMap`` output.
+The option is named ``inclusion`` and can take the following values: ``normal`` if no selection will apply, ``mapOnly`` if the SLD element should be used only during maps rendering and ``legendOnly`` if the element will be applied only when rendering legends eg.:
```

Comment:
The definition of what values do was specified twice.

```suggestion
The vendor option is named ``inclusion``, e.g.:
```

Reply:
fixed